src/credentials/CHIPCert.h

```diff
@@ -653,6 +653,24 @@ CHIP_ERROR ConvertX509CertToChipCert(const uint8_t * x509Cert, uint32_t x509Cert
 CHIP_ERROR ConvertX509CertsToChipCertArray(const ByteSpan & x509NOC, const ByteSpan & x509ICAC, uint8_t * chipCertArrayBuf,
                                            uint32_t chipCertArrayBufSize, uint32_t & chipCertBufLen);
 
+/**
+ * @brief Convert CHIP certificate array to Separate NOC and ICA Certificates.
+ *        This function takes a CHIP certificate array and splits it into single
+ *        NOC and ICA CHIP certificates.
+ *
+ *        NOC certificate must be provided.
+ *        ICA certificate is optional. It will be omitted (nullptr, 0) if the CHIP certificate array contains only one entry.
+ *
+ *        The API enforces that the NOC is issued by ICA (if ICA is provided).
+ *
+ * @param operationalCert Chip certificate array.
```

Comment:
Suggest renaming to opCertArray, since it's not just a single certificate.

Reply:
Will update.